markdown/epilogue.markdown

```diff
@@ -1,39 +1,151 @@
-If you are completely new to Git, we suggest you look at a beginner tutorial first (link - software carpentry http://software-carpentry.org/lessons/).
-What you will need to know to contribute to an open source project using Git (although some of this you can pick up by doing it!)
-- Basic Git commands: clone, add, commit, push, pull, branch.
-- Git commands that will likely be useful (but are not essential to get started):
-merge, rebase, cherry-pick.
-- Common GitHub actions: forking, pull requests, issues.
-
-Some resources for learning Git/GitHub:
-- Huff/Scopatz, Effective Computation in Physics
-- Other O'Reilly books on git/github/pull requests
-- Online tutorials
-
-### How to contribute to SciPy and related packages specifically (community rules?) and GitHub etiquette
-- When and where to raise GitHub issues
-- When to use pull requests
-
-### Testing
-
-## Open Science
-Why do open science?
-- Reminder about open source licenses (refer to introduction)
-- Open data
-- Open access and preprints
+
+### Mailing lists
+
+We mentioned in the preface that SciPy is a community. A great way to continue
+learning is to subscribe to the main mailing lists for NumPy, SciPy, pandas,
+matplotlib, scikit-image, and other libraries you might be interested in, and read
+them regularly.
+
+And when you do get stuck in your own work, don't be afraid to seek help there! We
+are a friendly bunch! The *main* requirement when seeking help is to show that
+you've tried a bit of problem solving yourself, and to provide others with a
+minimal script to demonstrate your problem and how you've tried to fix it.
+
+- **No:** "I need to generate a big array of random Gaussians. Can someone
+  help?"
+- **No:** "I have this huge library at https://github.com/ron_obvious, if you
+  look in the statistics library, there's a part that really needs random
+  Gaussians, can someone take a look???"
+- **Yes:** "I've been trying to generate a big list of random Gaussians like
+  so: `gauss = [np.random.randn()] * 10**5`. But when I compute `np.mean(gauss)`,
+  it's hardly ever as close to 0 as I expect. What am I doing wrong? The full
+  script ready for copy-paste is below.
+
+### GitHub
+
+We also talked in the preface about GitHub. All of the code that we discussed lives
+on GitHub:
+
+- https://github.com/numpy/numpy
+- https://github.com/scipy/scipy
+
+and others. When something isn't working as you expect, it could be a bug. If,
+after some investigation, you are convinced that you have indeed uncovered a
+bug, you should go to the "issues" tab of the relevant GitHub repository and
+create a new issue. This will ensure that the bug is on the radar of the
+developers of that library, and that it will (hopefully) be fixed in the next
+version. By the way, this advice also applies to "bugs" in the documentation:
+if something in a library's documentation isn't clear to you, file an issue!
+
+Even better than filing an issue is *submitting a pull request*. We can't cover
```

Comment:
And a documentation pull request is a great entry point as you're trying to gain confidence with the process.